gemhardware/managers/src/common/amc/AMCManager.cc

```diff
@@ -196,7 +202,41 @@ void gem::hw::amc::AMCManager::configureAction()
         amc->configureDAQModule(enableZS, doPhaseShift, runType, 0xfaac, m_relockPhase.value_, m_bc0LockPhaseShift.value_);
       } GEM_CATCH_RPC_ERROR("AMCManager::configureAction", gem::hw::devices::exception::ConfigurationProblem);
 
-      // FIXME everyting considering the scanning should be set by the gemcalibration module
+
+      if (m_scanInfo.bag.scanType.value_ == 2) {
```

Comment:
Can't `2` be replaced with a symbolic name for clarity?

Reply:
I think it is done to reflect the FW-based scan types, however I second that it should use the symbolic name, especially given that enums for this exist.